test/asserting/policy.py

```diff
@@ -77,9 +78,9 @@ def assertViolation(self, actual_violation, expected_violation):
 
         pprint(actual_violation)
 
-        assert actual_violation['name'] == expected_violation['name']
-        assert actual_violation['position'] == expected_violation['position']
-        assert actual_violation['level'] == expected_violation['level']
+        self.assertEqual(actual_violation['name'], expected_violation['name'])
+        self.assertEqual(actual_violation['position'], expected_violation['position'])
+        self.assertEqual(actual_violation['level'], expected_violation['level'])
```

Comment:
Why?
Aren't we using pytest here?

Reply:
The reason why I remove py.test specific code is the learning cost and the debugging cost. I feel that keep time for implementation shorter is important because I have only few moments to implement it.

For example, I forget sometimes how to use py.test, so the tests may got failure but understanding the error messages caused by misusing were not easy to me.

But I do not mean py.test is unuseful for all situation. Only this case, I determined that py.test is unnecessary.